config/prow/labels.yaml

```
@@ -234,6 +234,18 @@ repos:
         target: both
         prowPlugin: shrug
         addedBy: humans
+      - color: e11d21
+        description: Indicates the PR's author has not signed the cla-assistant.io CLA.
+        name: 'cla-assistant.io: no'
```

I would go for
```suggestion
        name: 'cla-assistant: no'
```

but that's only my personal preference.

---

Yep, `cla: yes` or `cla: no` sounds good.